Formula/assh.rb

```diff
@@ -1,9 +1,8 @@
 class Assh < Formula
   desc "Advanced SSH config - Regex, aliases, gateways, includes and dynamic hosts"
   homepage "https://github.com/moul/advanced-ssh-config"
```

Comment:
```suggestion
  homepage "https://manfred.life/assh"
```

GitHub repo suggests this as a homepage